packages/core/src/tracing/instrumentation/protocols/graphql.js

```diff
@@ -295,6 +295,9 @@ function finishWithException(span, err) {
 }
 
 function instrumentApolloGatewayExecuteQueryPlan(apolloGatewayExecuteQueryPlanModule) {
+  logger.warn(
```

Comment:
issue: We only want to log the warning when the customer is using `@apollo/federation`

Reply:
done